NEWS.md

```diff
@@ -1,6 +1,565 @@
+
+#### MXNet nGraph integration
+
+As the diversity of deep learning hardware accelerators increase, it is important to have an efficient abstraction layer so developers can avoid having to enable each accelerator/compute separately. Intel nGraph enables that vision. The primary goal of this integration is to provide a seamless development and deployment experience to data scientists and machine learning engineers to leverage Intel nGraph ecosystem with MXNet. As Subgraph API seamlessly integrates with MXNet frontend API, users should just be able to use or switch nGraph backend with any existing MXNet scripts, models and deployments using the symbolic interface. For more details see  [MXNet nGraph integration using subgraph backend interface](https://cwiki.apache.org/confluence/display/MXNET/MXNet+nGraph+integration+using+subgraph+backend+interface)
+
+#### JVM Memory Management
+
+MXNet Scala and Java API uses native memory to manage NDArray, Symbol, Executor, DataIterators using the MXNet c_api. C APIs provide appropriate interfaces to create, access and free these objects MXNet Scala has corresponding Wrappers and APIs which have pointer references to the native memory. Before this project, JVM users(Scala/Clojure/Java..) of Apache MXNet have to manage MXNet objects manually using the dispose pattern, there are a few usability problems with this approach:
+
+Users have to track the MXNet objects manually and remember to call dispose, this is not Java Idiomatic and not user-friendly, quoting a user "this feels like I am writing C++ code which I stopped ages ago"
+Leads to memory leaks if dispose is not called.
+Many Objects in MXNet-Scala are managed in native memory, needing to use dispose on them as well.
+bloated code with dispose() methods.
+hard to debug memory-leaks
+Goals of the project are to provide MXNet JVM Users automated memory management which can release native memory when there are no references to JVM objects, to be able to manage both GPU and CPU Memory automatically without performance degradation with automated memory management.  More details can be found here: [JVM Memory Management](https://cwiki.apache.org/confluence/display/MXNET/JVM+Memory+Management)
+
+#### Topology-aware AllReduce
```

Comment:
From what I have seen, and unless I have missed something, this feature is not working yet correctly. The test in p3.16xlarge are failing and we aren't able to partition the trees during C++ unit tests.  I would not announce it on the release notes.

https://github.com/apache/incubator-mxnet/issues/13341

I would only announce features which can be used by our users and don't waste their time and produce frustration.

Reply:
added note that it is an experimental feature and has known problems.